flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -411,66 +331,15 @@ class CDUAocOfpData {
     }
 }
 
-function setEstimatedBaggagePayload(mcdu) {
-    if (mcdu.aocWeight.fwdBag !== undefined || mcdu.aocWeight.rearBag !== undefined) {
-        return;
-    }
-
-    const currentPayload = getCurrentPayload(mcdu);
-
-    if (isNaN(currentPayload)) {
-        return;
-    }
-
-    const {forwardBaggageWeight, rearBaggageWeight} = getDistributedBaggagePayload(currentPayload, mcdu.aocWeight.zfwcg);
-
-    mcdu.aocWeight.estFwdBag = forwardBaggageWeight;
-    mcdu.aocWeight.estRearBag = rearBaggageWeight;
-}
-
-async function loadBaggagePayload(mcdu, updateView) {
-    const FORWARD_BAGGAGE_INDEX = 3 + 1; // Value from flight_model.cfg
-    const REAR_BAGGAGE_INDEX = 5 + 1; // Value from flight_model.cfg
-
-    const currentfwdBag = getCurrentFwdBag(mcdu);
-    const currentRearBag = getCurrentRearBag(mcdu);
-
-    mcdu.aocWeight.loading = true;
-    updateView();
-
-    const payloadCount = SimVar.GetSimVarValue("PAYLOAD STATION COUNT", "number");
-    for (let i = 1; i <= payloadCount; i++) {
-        switch (i) {
-            case FORWARD_BAGGAGE_INDEX: {
-                await SimVar.SetSimVarValue(`PAYLOAD STATION WEIGHT:${i}`, "kilograms", +currentfwdBag);
-                break;
-            }
-            case REAR_BAGGAGE_INDEX: {
-                await SimVar.SetSimVarValue(`PAYLOAD STATION WEIGHT:${i}`, "kilograms", +currentRearBag);
-                break;
-            }
-            default: {
-                await SimVar.SetSimVarValue(`PAYLOAD STATION WEIGHT:${i}`, "kilograms", 0);
-                break;
-            }
-        }
-    }
-
-    mcdu.aocWeight.loading = false;
-    updateView();
-
-    return;
-}
-
 async function loadFuel(mcdu, updateView) {
     const currentBlockFuel = mcdu.aocWeight.blockFuel || mcdu.simbrief.blockFuel;
 
     mcdu.aocWeight.loading = true;
     updateView();
 
-    const outerTankCapacity = 228; // Left and Right // Value from flight_model.cfg
-    const innerTankCapacity = 1816; // Left and Right // Value from flight_model.cfg
-    const centerTankCapacity = 2179; // Center // Value from flight_model.cfg
+    const outerTankCapacity = 228 + (1 * 2); // Left and Right // Value from flight_model.cfg (plus the unusable fuel capacity (GALLONS))
+    const innerTankCapacity = 1816 + (7 * 2); // Left and Right // Value from flight_model.cfg (plus the unusable fuel capacity (GALLONS))
+    const centerTankCapacity = 2179 + 6; // Center // Value from flight_model.cfg (plus the unusable fuel capacity (GALLONS))
```

Comment:
Not sure why this is being done as it would seem to de-sync the total weight from the MSFS weight and balance page? With SU5, MSFS has fixed the unusable fuel bug. We will be updating the flight_model.cfg file to use the following values when the engine fuel flow model is updated to not bug out when an unusable fuel value is present - Left/Right Aux tanks (i.e., outer tanks): 228 total/0 unusable, Left/Right main tanks (i.e., inner tanks): 1823.75 total/7.75 unusable, and Center tank: 2185.16 total/6.1 unusable.

Reply:
Hi Don! I think I must have confused from the time when I asked you about reverting the fuel capacity changes made to flightmodel.cfg.

This adjustment to the OFP page can be removed and we can go back to using the values you specified above (without the unusable capacity). In any case, we were planning in the dev chat to completely desync the aircraft from the MSFS weights and balance since there is still some issues in the sim and the way it processes those amounts. 

I'll update the values on the new PR #5547

Reply:
Oops, sorry, I made the comment against the wrong (old) PR. I don't remember exactly what was said before about reverting the fuel capacity changes. I'm hoping we can update the flight model.cfg file to the new numbers with unusable fuel soon.

Reply:
So from what I understood, the OFP data related adjustments can be reverted now, and we'll take a look at the flightmodel.cfg with the updated fuel model later?